### PR TITLE
Bug fix for manifest valid values validation

### PR DESCRIPTION
**Pull request by GiaJordan**

### CODE REVIEW

**Comment by allaway** [comment]
I will test this now, thank you!!

File: schematic/models/validate_manifest.py [hunk 1]
```diff
@@ -273,7 +273,7 @@ def validate_manifest_values(self, manifest, jsonSchema):
         
         errors = []
         warnings = []
-        annotations = json.loads(manifest.to_json(orient="records"))
+        annotations = json.loads(manifest.astype('string').to_json(orient="records"))
```
Comment (milen-sage) [inline schematic/models/validate_manifest.py:276]: yep - treating everything as string when loading in JSONSchema makes sense!

File: schematic/utils/df_utils.py [hunk 2]
```diff
@@ -34,7 +34,7 @@ def load_df(file_path, preserve_raw_input=True, **kwargs):
         float_df=deepcopy(org_df)
 
         #Find integers stored as strings 
-        ints = org_df.applymap(lambda x: np.int(x) if str.isdigit(x) else False, na_action='ignore').fillna(False)
+        ints = org_df.applymap(lambda x: int(x) if str.isdigit(x) else False, na_action='ignore').fillna(False)
```
Comment (milen-sage) [inline schematic/utils/df_utils.py:37]: @GiaJordan I know you switched to `np` for a reason - rolling back this particular line would work with your general approach on input going forward?

File: schematic/utils/df_utils.py [hunk 3]
```diff
@@ -34,7 +34,7 @@ def load_df(file_path, preserve_raw_input=True, **kwargs):
         float_df=deepcopy(org_df)
 
         #Find integers stored as strings 
-        ints = org_df.applymap(lambda x: np.int(x) if str.isdigit(x) else False, na_action='ignore').fillna(False)
+        ints = org_df.applymap(lambda x: np.int64(x) if str.isdigit(x) else False, na_action='ignore').fillna(False)
```
Comment (GiaJordan) [inline schematic/utils/df_utils.py:37]: @milen-sage update

**Comment by allaway** [comment]
I can't test in the DCA due to the other issues we have been discussing, but the manifest that I originally reported erros with seems to validate just fine w/CLI now! Thanks!

```
(scheamtic) ➜  schematic git:(develop-CSV-import) ✗ schematic model --config config.yml validate --manifest_path ~/Downloads/foo.csv -dt GenomicsAssayTemplate
Starting schematic...
The (model > input > location) argument with value '/Users/rallaway/Downloads/NF.jsonld' is being read from the config file.
The (model > input > file_type) argument with value 'local' is being read from the config file.
JSON schema successfully generated from schema.org schema!
JSON schema file log stored as /Users/rallaway/Downloads/NF.GenomicsAssayTemplate.schema.json
Profiler store is not configured; omitting it from active stores
Profiler store is not configured; omitting it from active stores
Profiler store is not configured; omitting it from active stores
Profiler store is not configured; omitting it from active stores
Profiler store is not configured; omitting it from active stores
/opt/miniconda3/envs/scheamtic/lib/python3.7/site-packages/jinja2/environment.py:1088: DeprecationWarning: 'soft_unicode' has been renamed to 'soft_str'. The old name will be removed in MarkupSafe 2.1.
  return concat(self.root_render_func(self.new_context(vars)))
	0 expectation(s) included in expectation_suite.
Calculating Metrics: 0it [00:00, ?it/s]
Your manifest has been validated successfully. There are no errors in your manifest, and it can be submitted without any modifications.
```

**Review by milen-sage — APPROVED** [review]

**Pull request merged**